vpath/pjtester.yaml

```
@@ -0,0 +1,10 @@
+pjConfigs:
+  prConfig: 
+    kyma-project:
+      third-party-images:
+        prNumber: 281
+  prowJobs:
+    kyma-project:
+      third-party-images:
+        - pjName: "pre-main-tpi-otel-collector"
+          report: true
```

prConfig inside pjConfigs is only used to change base prowjob definition PR
```suggestion
pjConfigs:
  prowJobs:
    kyma-project:
      third-party-images:
        - pjName: "pre-main-tpi-otel-collector"
          report: true
 prConfigs:
    kyma-project:
      third-party-images:
        prNumber: 281
```